packages/vite/src/node/server/transformRequest.ts

```diff
@@ -73,7 +73,7 @@ export function transformRequest(
   const pending = server._pendingRequests.get(cacheKey)
   if (pending) {
     return server.moduleGraph
-      .getModuleByUrl(removeTimestampQuery(url), options.ssr)
```

Comment:
Unrelated to the PR, the timestamp is already removed by `getModuleByUrl`. We can move it to a separate PR if it is too noisy here

Reply:
I wouldn't mind mixing it here as it's only few line changes, so I'm fine either ways 🙂